### DISCUSSION TITLE
Vue Auth error: Could not read password grant params

### DISCUSSION THREAD

**Discussion by samjay000** [supabase/supabase#6305] [Questions] [answered]
const {user, error} = await supabase.auth.signIn({
        email: email_,
        password: password_,
      })

Above code throws below error,
`Could not read password grant params: json: cannot unmarshal object into Go struct field PasswordGrantParams.email of type string`

**Comment by GaryAustin1** [comment]
Can you describe your environment, framework, local or hosted Supabase?  Is the code running on browser or server side?  Is password_ a string?  Does the environment have local storage?

**Comment by samjay000** [comment]
"@supabase/supabase-js": "^1.
"vue": "3.2.9",

Browser - firefox
Environment Supabase.io
Is password_ a string? Yes
I'm not using browser local storage

**Reply by GaryAustin1** [reply]
Near as I can tell it is an error in the format of the json object being sent to Supabase gotrue by your code or the Supabase client code.  
If you can see the network request going out and have the url and body it might help. 
The error is coming from here I believe:
https://github.com/supabase/gotrue/blob/ff4539b5bdaac044672111c6feaea3625b1ca751/api/token.go#L162
The object being json decoded is both the email and password.  
The error says email is not a string , but it could be that is the first item in the json.

**Comment by samjay000** [comment, marked as answer]
Thanks @GaryAustin1, I was passing ref not value.
